packages/scene-composer/public/animation-test-scene.scene.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
IS this a temporary setup?

Reply:
I don't think it should necessarily be temporary I think we should have a scene demonstrating animations but I might be wrong